### ISSUE TITLE
Google login error

### ISSUE THREAD

**Issue by athulkrishna2015** [Zren/plasma-applet-eventcalendar#352]
Access blocked: KDE Event Calendar Widget’s request is invalid

I am getting this error
![image](https://github.com/Zren/plasma-applet-eventcalendar/assets/35728672/038a60c4-7a6f-4da2-9acc-9f4b07c82fbf)

**Comment by srsgores** [comment]
Yeah, I get this too.  I think the author needs to [create a new google console app](https://github.com/Zren/plasma-applet-eventcalendar/blob/master/package/contents/ui/calendars/GoogleApiSession.qml#L62) :arrow_backward: needs to be updated along with the client secret.

**Comment by Tintow** [comment]
Getting the same error.  Error details are

*Error 400: invalid_request
The out-of-band (OOB) flow has been blocked in order to keep users secure. Follow the Out-of-Band (OOB) flow migration guide linked in the developer docs below to migrate your app to an alternative method.
Request details: redirect_uri=urn:ietf:wg:oauth:2.0:oob*

**Comment by jambolo1970** [comment]
same problem
`Errore 400: invalid_request
The out-of-band (OOB) flow has been blocked in order to keep users secure. Follow the Out-of-Band (OOB) flow migration guide linked in the developer docs below to migrate your app to an alternative method.
Richiedi dettagli: redirect_uri=urn:ietf:wg:oauth:2.0:oob `

**Comment by nicc777** [comment]
Really looking forward to this issue being fixed

**Comment by amsesk** [comment]
Same error here - would really like to use the widget!

**Comment by racingbrett** [comment]
also getting this error..
if it helps
Error 400: invalid_request
The out-of-band (OOB) flow has been blocked in order to keep users secure. Follow the Out-of-Band (OOB) flow migration guide linked in the developer docs below to migrate your app to an alternative method.
Request details: redirect_uri=urn:ietf:wg:oauth:2.0:oob

**Comment by TheRicePirate** [comment]
Same error here

**Comment by gardotd426** [comment]
Is this project dead or what?

**Comment by joaociocca** [comment]
> Is this project dead or what?

I'm affected with this issue and was looking forward to using Google Calendar integration, but I wouldn't call something updated last August as dead.

you can use this to workaround the problem > https://github.com/Zren/plasma-applet-eventcalendar/pull/345#issuecomment-1584348515

**Comment by ALikesToCode** [comment]
The project unfortunately seems dead to me, for future references, you can clone from https://github.com/ALikesToCode/plasma-applet-eventcalendar, I made sure that it is most update to date fork for now.

**Comment by balaramadurai** [comment]
> The project unfortunately seems dead to me, for future references, you can clone from https://github.com/ALikesToCode/plasma-applet-eventcalendar, I made sure that it is most update to date fork for now.

Thanks @ALikesToCode, your fork works very well. Thanks a lot for taking this up and doing this.

Thanks @Zren for the original code. Excellent extension!

**Comment by Struki84** [comment]
> The project unfortunately seems dead to me, for future references, you can clone from https://github.com/ALikesToCode/plasma-applet-eventcalendar, I made sure that it is most update to date fork for now.

hey @ALikesToCode, tnx for the fork. I've tried the install script but it fails `desktoptojson: not found` I cant seem to find the packages that install desktoptojson. 

I'm running popos 22.04 with KDE 5.24.7. I've also installed `qttools5-dev-tools` with no avail. Any ideas? is there any other way to install your corrected fork?
